blocks/newsletter-subscribe/newsletter-subscribe.css

```diff
@@ -0,0 +1,371 @@
+.newsletter-subscribe.block {
+  padding: 0 39px;
+  max-width: 1440px;
+  margin: 0 auto;
+
+  --circle-svg: url("data:image/svg+xml,%3Csvg%20viewBox%3D%220%200%20100%20100%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%3E%3Ccircle%20fill%3D%22%23fff%22%20cx%3D%2250%22%20cy%3D%2250%22%20r%3D%2250%22%20%2F%3E%3C%2Fsvg%3E");
+}
+
+.newsletter-subscribe.block .subscribe-form {
+  display: flex;
+  flex-direction: column;
+  gap: 30px;
+}
+
+.newsletter-subscribe.block .subscribe-form a:hover {
+  color: inherit;
+}
+
+.newsletter-subscribe.block .subscribe-form label[for*="checkbox"] {
+  cursor: pointer;
+}
+
+.newsletter-subscribe.block .subscribe-form input.toggle {
+  cursor: pointer;
+  width: 51px;
+  height: 30px;
+  border-radius: 15px;
+  margin: 0;
+  appearance: none;
+  background: var(--circle-svg), #e1e1e3;
+  background-size: 22px 22px;
+  background-repeat: no-repeat;
+  background-position: 4px 4px;
+  transition: background 0.4s ease;
+}
+
+.newsletter-subscribe.block .subscribe-form input.toggle:checked {
+  background: var(--circle-svg), #1f2022;
+  background-size: 22px 22px;
+  background-repeat: no-repeat;
+  background-position: 26px 4px;
+}
+
+.newsletter-subscribe.block .subscribe-form .title-wrapper {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.newsletter-subscribe.block .subscribe-form .title-wrapper h1,
+.newsletter-subscribe.block .subscribe-form .title-wrapper h2,
+.newsletter-subscribe.block .subscribe-form .title-wrapper h3 {
+  font-family: var(--font-tungsten);
+  font-size: 67px;
+  font-weight: 600;
+  line-height: 66px;
+  letter-spacing: 0.05em;
+  text-transform: uppercase;
+  color: black;
+  margin: 0 0 10px;
+}
+
+.newsletter-subscribe.block .subscribe-form .title-wrapper [slot="description"] p {
+  font-family: "Times New Roman", Times, serif;
+  text-align: center;
+  max-width: 500px;
+  font-style: normal;
+  font-weight: 325;
+  font-size: 18px;
+  margin: 0;
+  color: #4e515e;
+  line-height: 30px;
+}
+
+.newsletter-subscribe.block .subscribe-form .title-wrapper [slot="tos-top"] {
+  color: #1f2022;
+  text-align: center;
+  font-family: "Times New Roman", Times, serif;
```

Comment:
Can you move that font into a CSS variable (like you did with `--circle-svg`) ?

Reply:
I created a times variable.
It might also be worth mentioning, on the original golf site, they probably didn't intend to use times now roman, but something went wrong with the font definition.
I opted to copy it as it appears though since I would assume if it was a problem GD would have fixed it by now.
![image](https://github.com/headwirecom/helix-sportsmagazine/assets/68734204/e88d37e1-dc57-4b34-a205-597982a8d559)